.gitignore

```diff
@@ -6,7 +6,7 @@ composer.lock
 .libs
 .deps
 **.code-workspace
-Makefile*
+/Makefile*
```

Comment:
What's the point of this change?

Reply:
Its not related to this PR. But previous rule also caught Makefile it wasn't supposed to. I'll remove this, since its also present in Laravel PR and sneaked in here.